### PR TITLE
adis16480 accel and gyro reads only

### PR DESCRIPTION
**Pull request by jpineda3**
I added a switch case in stepImpl that would either call stepAccelGyro or stepAccelGyroMag depending on number of EnabledChannels.

setScales was modified to return only the scales for the enabled channels, otherwise matrix dimensions would not agree.

**This PR does not address the issue with data size mismatch in the buffer.**

Added examples based on existing adis16460 scripts.

### CODE REVIEW

File: +adi/ADIS16480.m [hunk 1]
```diff
@@ -56,7 +56,14 @@
         
         function [accelReadings, gyroReadings, magReadings, valid] = stepImpl(obj)
             [dataR, valid] = stepImpl@adi.common.Rx(obj);
-            [accelReadings, gyroReadings, magReadings] = stepAccelGyroMag(obj, dataR);
+            switch length(obj.EnabledChannels)
+                case 6
+                    [accelReadings, gyroReadings] = stepAccelGyro(obj, dataR);
+                    magReadings = null(obj.SamplesPerRead,3);
```
Comment (tfcollins) [inline +adi/ADIS16480.m]: I don’t think you want null here, use varargout out  for variable size outputs

Comment (jpineda3) [reply +adi/ADIS16480.m]: I implemented varargout here: https://github.com/jpineda3/SensorToolbox/blob/1e816bacb7e49b1af8154e660bb0103c26e73133/%2Badi/ADIS16480.m#L67

This errors "Output argument "varargout{2}" (and maybe others) not assigned during call to "adi/ADIS16480/stepImpl"."

File: +adi/ADIS16480.m [hunk 2]
```diff
@@ -38,25 +38,37 @@
         end
     end   
     
-    methods (Access=protected)
-        function numOut = getNumOutputsImpl(~)
-            numOut = 4;
-        end       
-    end
+%     methods (Access=protected)%REMOVED WHILE stepImpl USES varargout
```
Comment (jpineda3) [inline +adi/ADIS16480.m]: Addressed this [error](https://github.com/analogdevicesinc/SensorToolbox/pull/10#discussion_r748945441) by removing getNumOutputsImpl definition here. stepImpl and read method now return variable number of output depending on number of enabled channels.

Comment (tfcollins) [reply +adi/ADIS16480.m]: That works. Please remove the comments code :)

File: sensor_examples/adis16480_motion.m [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+IMU.uri = 'ip:analog';
+IMU.SampleRate = 128;
+IMU.EnabledChannels = 1:6;
+fs = IMU.SampleRate;
+% Filter
+ifilt = imufilter('SampleRate', fs);
+% Scopes
+N = 500;
+viewer = HelperOrientationViewer;
+% ts = dsp.TimeScope;
```
Comment (tfcollins) [inline sensor_examples/adis16480_motion.m]: Can you change the commented code to be uncommented and surrounded by a conditional? Less work if people want to enabled the scopes.

Comment (jpineda3) [reply sensor_examples/adis16480_motion.m]: I added a useScope flag to enclose the use of timescope in a conditional. useScope is true by default. The adis16460 example is updated as well.

**Pull request merged**